tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -1840,6 +1785,64 @@ def test_task_output_json_schema(
         self.assertEqual(exp_granules, result_value)
         mock_post_entry_to_queue.assert_called_once()
 
+    @patch.dict(
+        os.environ,
+        {
+            request_files.OS_ENVIRON_ORCA_DEFAULT_GLACIER_BUCKET_KEY: uuid.uuid4().__str__()
+        },
+        clear=True,
+    )
+    def test_get_default_glacier_bucket_name_returns_override_if_present(self):
+        bucket = Mock()
```

Comment:
Not sure why we are using Mock() here for bucket when this in reality is a string not an object. This should really just be a known string throughout these tests I would think. Thoughts?

Reply:
I like mocks as a sort of added check that no processing is done with them. There shouldn't be any processing here, so this raises an error if, say, the order of checks is wrong.

Reply:
Side-pedantic-note, strings in Python are objects. :]

Reply:
Good point